### ISSUE TITLE
Classifying States of Cooking Objects Using Convolutional Neural Network. (arXiv:2105.14196v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7448]
Classifying States of Cooking Objects Using Convolutional Neural Network. (arXiv:2105.14196v1 [cs.CV])<br>
https://ift.tt/3i6jH93<br>
<p>Automated cooking machine is a goal for the future. The main aim is to make the cooking process easier, safer, and create human welfare. To allow robots to accurately perform the cooking activities, it is important for them to understand the cooking environment and recognize the objects, especially correctly identifying the state of the cooking objects. This will significantly improve the correctness of the following cooking recipes. In this project, several parts of the experiment were conducted to design a robust deep convolutional neural network for classifying the state of the cooking objects from scratch. The model is evaluated by using various techniques, such as adjusting architecture layers, tuning key hyperparameters, and using different optimization techniques to maximize the accuracy of state classification.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
